ydb/core/grpc_services/rpc_view.cpp

```diff
@@ -0,0 +1,94 @@
+#include "rpc_scheme_base.h"
+#include "service_view.h"
+
+#include <ydb/core/grpc_services/base/base.h>
+#include <ydb/core/tx/schemeshard/schemeshard.h>
+#include <ydb/core/ydb_convert/ydb_convert.h>
+#include <ydb/library/actors/core/actor.h>
+#include <ydb/library/actors/core/hfunc.h>
+#include <ydb/public/api/protos/draft/ydb_view.pb.h>
+
+namespace NKikimr::NGRpcService {
+
+using namespace Ydb;
+
+using TEvDescribeView = TGrpcRequestOperationCall<View::DescribeViewRequest, View::DescribeViewResponse>;
+
+class TDescribeViewRPC : public TRpcSchemeRequestActor<TDescribeViewRPC, TEvDescribeView> {
+    using TBase = TRpcSchemeRequestActor<TDescribeViewRPC, TEvDescribeView>;
+
+public:
+    using TBase::TBase;
+
+    void Bootstrap() {
+        DescribeScheme();
+    }
+
+    void PassAway() override {
+        TBase::PassAway();
+    }
+
+private:
+    void DescribeScheme() {
+        auto ev = std::make_unique<TEvTxUserProxy::TEvNavigate>();
+        SetAuthToken(ev, *Request_);
+        SetDatabase(ev.get(), *Request_);
+        ev->Record.MutableDescribePath()->SetPath(GetProtoRequest()->path());
+
+        Send(MakeTxProxyID(), ev.release());
```

Comment:
If you are wondering, all describe requests sent to tx proxy [are going](https://github.com/ydb-platform/ydb/blob/56fbcfc12d0e9b60d405a93e782ffcb084376446/ydb/core/tx/tx_proxy/proxy_impl.cpp#L407) through both:
- [scheme cache](https://github.com/ydb-platform/ydb/blob/56fbcfc12d0e9b60d405a93e782ffcb084376446/ydb/core/tx/tx_proxy/describe.cpp#L293) ([with](https://github.com/ydb-platform/ydb/blob/56fbcfc12d0e9b60d405a93e782ffcb084376446/ydb/core/tx/tx_proxy/describe.cpp#L282) sync version flag) to retrieve the path's SchemeShard id
- [SchemeShard](https://github.com/ydb-platform/ydb/blob/56fbcfc12d0e9b60d405a93e782ffcb084376446/ydb/core/tx/tx_proxy/describe.cpp#L391) to retrieve the most up-to-date info about the path